src/main/java/teammates/common/datatransfer/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -200,17 +187,18 @@ public String getQuestionResultStatisticsHtml(
             String name = bundle.getNameForEmail(participantIdentifier);
             String teamName = bundle.getTeamNameForEmail(participantIdentifier);
             
-            fragments.append(Templates.populateTemplate(FeedbackQuestionFormTemplates.RANK_RESULT_STATS_RECIPIENTFRAGMENT,
-                                                                        "${rankOptionValue}", Sanitizer.sanitizeForHtml(name),
-                                                                        "${team}", Sanitizer.sanitizeForHtml(teamName),
-                                                                        "${ranksReceived}", ranksReceived,
-                                                                        "${averageRank}", df.format(average)));
+            fragments.append(Templates.populateTemplate(FormTemplates.RANK_RESULT_STATS_RECIPIENTFRAGMENT,
+                                                                        Slots.RANK_OPTION_VALUE,
+                                                                        Sanitizer.sanitizeForHtml(name),
```

Comment:
Looks like I missed this before. Either indent this one by 8 more spaces, or shift this whole bunch to the left (so `Slots.*` starts from 8 spaces after `[f]ragments`) so that this can be on the same line.